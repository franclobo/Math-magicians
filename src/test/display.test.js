import renderer from 'react-test-renderer';
import Display from '../components/display';

it('renders correctly', () => {
  const tree = renderer
    .create(<Display />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
