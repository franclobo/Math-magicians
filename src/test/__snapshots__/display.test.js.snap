// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="display"
>
  <span
    className="previous"
  />
  <span
    className="operand"
  />
  <span
    className="current"
  >
    0
  </span>
</div>
`;
